pyscraper/sp/parse-official-reports-new.py

```diff
@@ -666,7 +701,7 @@ def parse_html(session, report_date, soup, page_id, original_url):
                     elif maybe_time:
                         current_time = maybe_time
                     else:
-                        if current_votes:
+                        if current_votes and speech_part != "\n":
```

Comment:
Would it be better to catch a space-only speech_part higher up and continue? Looks like nothing will be added as tidied_paragraph will be empty.